pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/Dispatcher.java

```diff
@@ -40,8 +40,18 @@
 
     boolean canUnsubscribe(Consumer consumer);
 
+    /**
+     * disconnect all consumers and mark dispatcher closed to stop new incoming requests
+     * 
+     * @return
+     */
     CompletableFuture<Void> disconnect();
 
+    /**
+     * mark dispatcher open to serve new incoming requests
+     */
+    void connect();
```

Comment:
Naming sound a bit strange to me, since the the dispatcher itself does not "connect" to anything :) 

What about `reset()` or something else?